Change markdown-content hr style #5987

KentShikama opened the pull request:
The hr tag in the markdown-content was #eee,
and thus almost invisible. It has been changed to
$border-gray (currently #ddd) to be consistent
with the other hr tags.

Commit:
Change markdown-content hr style #5987

The <hr> tag in the markdown-content was #eee,
and thus almost invisible. It has been changed to
$border-gray (currently #ddd) to be consistent
with the other <hr> tags.

jhass commented:
I think I'd like it a bit more distinct from our usual borders surrounding the post, how about something like a width of 85% and centering it?

theworldbright commented:
What reason is there for having no margin after the last paragraph element in a post?

#main_stream .stream_element > .media .post-content p:last-of-type {
  margin-bottom: 0;
}

jhass commented:
Because it looks better?

Commit:
Adjust markdown-content hr style further

The hr tag is now centered at 85% width. It also
adds a default vertical margin that is equivalent
to the 0.8em the paragraph tags provide.

theworldbright commented:
Yeah I guessed as much. I originally had a problem with the way the vertical margins after last paragraph elements were working because of that specific style but I found a way around it.

The following markup

```
# Heading of post
<hr>
The title and this line have been separated with a hr tag
<hr>
This line and the line above have been separated with a hr tag
<hr>
A longer paragraph text text text text text text text text text text text text text text text text text text text text text text text text text text text text end of paragraph
```
now produces

![screen shot 2015-06-01 at 1 06 11 am](https://cloud.githubusercontent.com/assets/6329898/7902641/8c7bda48-07fa-11e5-97ff-b99d05728383.png)

Commit:
Refactor scss for markdown-content hr

jhass commented:
Merged as 45a5e3d1c5b76b8d83caf3bfec730cdbab7bdae2

Thank you!
